### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by sgurukrupa** [tldraw/tldraw-v1#546] [Q&A] [answered]
Hi,

Apart from this [link](https://github.com/tldraw/tldraw/blob/main/guides/documentation.md), is there any documentation on @tldraw/core and @tldraw/tldraw APIs?

Regards.

**Comment by steveruizok** [comment, marked as answer]
Not yet! We'll be creating proper docs for the new version ahead of our open source release.
